src/code/sys_ucode.c

```diff
@@ -1,9 +1,20 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/sys_ucode/SysUcode_GetUCodeBoot.s")
+u64* D_801D1E60 = D_801E2160;
+u64* D_801D1E64 = D_801E3790;
```

Comment:
Are these the start and end offsets of the ucode? If so we can document this data.